lua/simple-note.lua

```diff
@@ -63,17 +67,30 @@ M.listNotes = function()
   })
 end
 
-M.createAndOpenNoteFile = function()
-  local full_path = M.createNoteFile()
+M.createAndOpenNoteFile = function(opts)
+  local full_path = M.createNoteFile(opts)
+
   vim.cmd("edit " .. full_path)
   vim.notify(full_path .. " has been created")
```

Comment:
```suggestion
if full_path ~= nil then
  vim.cmd("edit " .. full_path)
  vim.notify(full_path .. " has been created")
end
```
 to avoid printing the success message when the note creation failed